### DISCUSSION TITLE
How to run the result by LS-PrePost?

### DISCUSSION THREAD

**Discussion by Majid-MHZ** [OpenRadioss/OpenRadioss#2736] [Q&A]
Hello everyone,
I hope you are doing well.

I created a .k file using LS-PrePost and ran it through OpenRadioss, which produced a .vtk file to visualize the result. I'm looking to create an output file that can be used in LS-PrePost. Do you have any suggestions on how to accomplish this?

Best regards,
Majid

**Comment by MarianBulla** [comment]
Hello @Majid-MHZ,
thanks for you interest in OpenRadioss.
Please check the discussion board for this.
Especially here: https://github.com/orgs/OpenRadioss/discussions/2361

Hope it helps you ?

Thanks and best regards,
Marian
